process/transaction/metaProcess.go

```diff
@@ -94,14 +92,6 @@ func (txProc *metaTxProcessor) ProcessTransaction(tx *transaction.Transaction) (
 
 	err = txProc.checkTxValues(tx, acntSnd, acntDst)
 	if err != nil {
-		if errors.Is(err, process.ErrUserNameDoesNotMatchInCrossShardTx) {
```

Comment:
do we still need the process.ErrUserNameDoesNotMatchInCrossShardTx error?

Reply:
yes. but not in metachain.

Reply:
ok